gr-blocks/grc/blocks_file_sink.block.yml

```diff
@@ -52,7 +52,7 @@ templates:
 cpp_templates:
     includes: ['#include <gnuradio/blocks/file_sink.h>']
     declarations: 'blocks::file_sink::sptr ${id};'
-    make: 'this->${id} = blocks::file_sink::make(${type.size}*${vlen}, ${file}, ${append});'
+    make: 'this->${id} = blocks::file_sink::make(${type.size}*${vlen}, "no_quotes(${file})", ${append});'
```

Comment:
I'm getting an error here with the file sink:
```
file_sink_base :error: [::open] no_quotes('/tmp/asdf.txt'): No such file or directory
```

Reply:
Sorry for that silly mistake of mine. I tested that, but pushed a wrong commit.
I fixed that now.